### ISSUE TITLE
Due May 28th: Redirect Request: Legacy Yellow Ribbon Pages

### ISSUE THREAD

**Issue by Karak888** [department-of-veterans-affairs/va.gov-team#4975]
### Instructions
- Requests for redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders of pending redirect.  

### Description
(Update May 1, 2020
We will redirect CURRENT year pages for 2019-2020 to the tool when tool release is complete.
URLs that were provided in comments previously are now in the main description of this ticket.

### Requestor Info
*VSA Public Websites*

### Implementation Date
Date new URL(s) will be live: TBD, launch day for YR Tool- AFTER tool is live on production

Please indicate when the redirect(s) needs to be implemented"
- [X] On the same date the new URLs launch
- [x] Can happen within 1-5 business days after new URLs launch
- [ ] Other - Please indicate timing : 

**The requesting team is responsible for notifying the group working on this issue if the target date changes**

### Redirects Needed
(UPDATE May 1, 2020)
here's the list of all the ~legacy~ current 2019-2020 YR schools pages, which will redirect to the tool when it is launched

Current URL  |  Redirect to new URL
---  |  ---
www.benefits.va.gov/gibill/yellow_ribbon/yrp_list_2019.asp <br> **Update this existing redirect** | https://www.va.gov/education/yellow-ribbon-participating-schools/
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ak.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/al.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ar.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/az.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ca.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/co.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ct.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/dc.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/de.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/fl.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ga.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/hi.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ia.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/id.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/il.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/in.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ks.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ky.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/la.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ma.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/md.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/me.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mi.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mn.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mo.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ms.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mt.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nc.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nd.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ne.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nh.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nj.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nm.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nv.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ny.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/oh.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ok.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/or.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/overseas.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/pa.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/pr.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ri.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/sc.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/sd.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/tn.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/tx.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ut.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/va.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/vt.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wa.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wi.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wv.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*
*https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wy.asp* | *https://www.va.gov/education/yellow-ribbon-participating-schools/*

### Steps to complete
- [ ] Above information is provided and issue is tagged and assigned appropriately - *@ requesting team*
- [ ] All appropriate VA stakeholders are notified of pending redirect - *@ requesting team*
- [x] Redirect request is vetted and documented and implementation plan is clear - *@ Content & IA team*
- [x] Request is assigned to appropriate team for implementation - *@ Content & IA team*
- [ ] Implementation team completes work - *@ Implementation team*
- [ ] Implementation pushed live and redirect is validated in production - *@ all*
- [ ] Ticket is closed - *@ requesting team*

**Comment by mnorthuis** [comment]
@andaleliz There are a number of legacy URLs involved in the YR schools pages and since the top level  legacy YR page is already redirected, I can't find all the child pages and URLs.   I believe there was a top level page with the state listing and links to historical info.   We will need to identify all of those URLs so we can determine how/if they should be retired.

**Comment by andaleliz** [comment]
@mnorthuis To be clear, am I responsible for identifying all of those URLs, or is that a task for a content resource?

The accordions on [this page](https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/#does-my-school-participate-in-) contain about 50 URLs that will need to redirect to our tool. One approach could be to redirect all to the tool landing page; another option could be to create a unique redirect w/ the search parameter so if a user hits the [URL for Florida schools](https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/fl.asp), they get to a search result page w/ only results in Florida.

**Comment by mnorthuis** [comment]
I'm not sure who is best equipped to get the URLs, I can dig around and see if I can figure out how to access them, but we may need assistance from someone on the Education side that handled those legacy pages. 

Do the legacy pages have significant SEO value for each individual state?  If they don't, it may be easiest for us to just redirect all state pages to the main page of the new tool.

**Comment by andaleliz** [comment]
I don't know how to definitively answer the SEO value/state question - I did some google searches with the following results:

- `california yellow ribbon schools` returned the [state list/map page for 2018](https://www.benefits.va.gov/GIBILL/yellow_ribbon/yrp_list_2018.asp) at the top of the search

- searching for `maryland`, `south carolina`, and `texas` + `yellow ribbon schools` returned the specific state page for the 2018-2019 school year for those states

Seems like Google returns outdate info for state specific searches, which isn't ideal.  I'm hesitant to make a recommendation without someone w more analytics expertise (who should I tag?) weighing in.

**Comment by mnorthuis** [comment]
I'll do a bit a digging on this.  Can you confirm whether we the historical pages need to remain findable or if they can be redirected?

**Comment by andaleliz** [comment]
I can't - @jenniferlee-dsva, what are your thoughts?

**Comment by jenniferlee-dsva** [comment]
@AnneHurley - Can you please add the URLs of the 50 (or so) static state YR tables to the original ticket above?  

The tables themselves will be - based on @mnorthuis's IA recommendation -- _either_ redirected to:
the new tool page or to the YR program page. 

FYSA @AkucherGCIO @lunascoop :  These redirects can happen shortly following the tool launch, if coordinating the timing is difficult to do with tool launch. Table redirects are not a dependency for the tool launch.

**Comment by AnneHurley** [comment]
Current URL | Redirect to new URL
-- | --
enter current URL | enter new URL

**Comment by AnneHurley** [comment]
@jenniferlee-dsva @mnorthuis @andaleliz  
Slack thread today with Mikki and Liz:
Mikki Northuis:spiral_calendar_pad:  18 days ago
@Liz Lantz I have a list of legacy Yellow Ribbon URLs and a proposed redirect approach, but I do have a couple questions that we will need to answer:
Historical yellow ribbon pages go back to 2009, do we need to provide links anywhere in the experience to some/all of the historical lists?
If yes, we will need to figure out what to do with the 2019/20 data when the 2020/21 data comes out this summer so we can link to it
There is one page that seems to be geared towards school administrators that needs to be handled.  I will follow up with the BAH team on possibly adding a link to it from the hub page they created for school administrators (https://www.benefits.va.gov/GIBILL/yellow_ribbon/Yellow_Ribbon_Info_Schools.asp)
(edited)

Liz Lantz  18 minutes ago
I completely missed this! My apologies. Thank you for resurfacing this today @Mikki.
In our meeting on 12/10/2019, Tammy said we do not need to keep the historical links.
cc @Anne Hurley and @Aruna Kucherlapati

So to recap we need the answer to the school administrator page question/BAH.

**Comment by mnorthuis** [comment]
@AnneHurley I have talked with BAH and the Education stakeholders will determine if a link to that page needs to be added to the school administrators hub page - either way, that content will not be impacted here as it is not Veteran content.

**Comment by AnneHurley** [comment]
@andaleliz  @jenniferlee-dsva  @kelsonic  @AkucherGCIO  Thanks to Kelson, here's the list of all the legacy YR schools pages, which will redirect to the YR main page where the YR tool lives, or to the YR tool itself - TBD.

// Execute in your terminal to open all of the below pages:
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ak.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/al.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ar.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/az.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ca.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/co.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ct.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/dc.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/de.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/fl.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ga.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/hi.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ia.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/id.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/il.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/in.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ks.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ky.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/la.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ma.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/md.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/me.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mi.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mn.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mo.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ms.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/mt.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nc.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nd.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ne.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nh.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nj.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nm.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/nv.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ny.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/oh.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ok.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/or.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/overseas.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/pa.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/pr.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ri.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/sc.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/sd.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/tn.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/tx.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/ut.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/va.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/vt.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wa.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wi.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wv.asp"
open "https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/wy.asp"

**Comment by mnorthuis** [comment]
@AkucherGCIO Will we be redirecting 2009 through 2019?  If so, I think we can I can work with Nick on this without having a full list of state pages for every year.

**Comment by AnneHurley** [comment]
@andaleliz  - Liz I believe you said that Tammy and team do not want us to keep any of the legacy YR rates page, correct? @mnorthuis  @AkucherGCIO  FYSA

Also, @jenniferlee-dsva - Is it worth making a style/content rule about when we *do* keep legacy rates, how far back is it necessary to go? We've discussed keeping the last 5 years max in general, but maybe this is a situation where it depends on the type of benefit?

**Comment by andaleliz** [comment]
@AnneHurley - Tammy did say that in the meeting we all attended on 12/10.  However, I just looked further down the notes at the [wrap up section](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/public-websites/yellow-ribbon-mvp/research/dec-2019-sme-discovery.md#wrap-up), and the group commented that
> need to get buy-in from their leadership team to agree to any changes we propose. Leadership at least needs to be aware and say this is what we're proposing and this is where we'd like to go
and also goes on to mention someone named Tom
>Tammy thinks it'd be Tom and those guys, who would be worried about old data. Tammy personally doesn't know how important that old data is to the YR folks. She doesn't know if they go back and look at old things, doesn't know what matters to them

So, we should double check with that team on this point.  The last time I spoke to them, we weren't close to discussing redirects or plans for the old data.  When we first started this work, @jenniferlee-dsva asked that you be the POC to interface w/ the SME - can you please reach out to them and inquire about this?

**Comment by AnneHurley** [comment]
@andaleliz  and @mnorthuis  - When we rewrote the main Yellow Ribbon page last year, we did not import over the historic rates tables. So our current page doesn't have those rates: https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/

You also can't get to them by each individual state  on the legacy state pages. 

The only place that I can find them is on this legacy Yellow Ribbon page: https://www.benefits.va.gov/gibill/yellow_ribbon/yellow_ribbon_info_schools.asp . @mnorthuis Is that the school-oriented page you said we didn't need to worry about per the BAH team? 

Since we don't have the historic charts on our current page or in the accordions, and the legacy state pages also don't have them, I'm wondering if that decision's already been made. @jenniferlee-dsva  - thoughts? Would we assume we aren't keeping historic YR data because we currently already aren't; or, if we think we do need to keep them, should we consider adding an accordion in the bottom Q&A section, *What are the Yellow Ribbon rates for previous years?*

**Comment by jenniferlee-dsva** [comment]
The page you found @AnneHurley is for `school administrators`, so we would NOT redirect that page ever for this YR tool effort: https://www.benefits.va.gov/gibill/yellow_ribbon/yellow_ribbon_info_schools.asp

The student page where this historic info lived can be found on the Internet Archive here (before it was redirected to our YR program page): https://web.archive.org/web/20190111021555/https:/www.benefits.va.gov/GIBILL/yellow_ribbon/yrp_list_2018.asp  (Scroll to the bottom of pg to see prior years tables.)

Per Liz's comment ^ - since Tammy was not able to provide a definitive answer regarding the need for historic YR schools, I recommend emailing Tammy, Luke, (and please cc Matt Self and me) that we need them to confirm the status of these prior year school lists, and that we are recommending: 

**A. If the prior year tables `are not `legally required**
- redirect those prior years historical list of YR schools to the new YR schools tool.

**B. If the prior year tables `are `legally required to be provided `to students`**
- we will link from our YR program page to this page with the historical info: https://www.benefits.va.gov/gibill/yellow_ribbon/yellow_ribbon_info_schools.asp [or whatever historical index page they propose]

**C. If the prior year tables are required but `only for school administrators`**
- we will NOT redirect the old tables, and we will not not link to them from our YR program or tool page. 

@AkucherGCIO - if there's more spin, let's tb on a call.

**Comment by AnneHurley** [comment]
@jenniferlee-dsva Will reach out to Tammy and team.

**Comment by AkucherGCIO** [comment]
@AnneHurley @mnorthuis - If there are more questions, I am happy to set up a call with you both, myself and Jen.  Please let me know.

**Comment by andaleliz** [comment]
@AnneHurley re your last comment, will you please cc me when you send that so I can be in the loop? thanks!

**Comment by AnneHurley** [comment]
@andaleliz  - turns out that @jenniferlee-dsva  spoke with Tammy and team in the weekly VBA meeting, and Tammy said no historic rates tables necessary. Apologies you weren't on that thread.

**Comment by andaleliz** [comment]
@AnneHurley yahoo! Thanks for the update :)

**Comment by mnorthuis** [comment]
Waiting to get final direction on this from SMEs and Jen.

**Comment by jenniferlee-dsva** [comment]
Adding update here, as we've been mainly Slacking progress updates. 

1/ Edu Deputy Director is reviewing the tool demo video this week. I will follow up with them next week to confirm outcome of that.

2/ We have changed the proposal to Janice, Joe, Tammy and Luke - regarding prior years: 
- The historic school years will continue to live on the legacy page (here: https://www.benefits.va.gov/gibill/yellow_ribbon/yellow_ribbon_info_schools.asp ), **but we will not link to historic information from the modernized experience.** Therefore we can't redirect historic years information. 

The above is a compromise bc both Tammy and Joe have confirmed that prior years info is not needed by students or by regulation. Janice feels strongly that prior years info must continue to be provided.  This compromise lets them continue to maintain prior years info on their legacy pages, without us linking to them.

After the first year of the modernized tool experience, there will be no more prior year TeamSite historic pages (for instance, for 2020-21 school year).  If Janice needs to continue providing previous year info after 2020-21, the YR team will need to do so as a GIDS database export that's made into a PDF and hosted on their legacy page: https://www.benefits.va.gov/gibill/yellow_ribbon/yellow_ribbon_info_schools.asp

**Comment by mnorthuis** [comment]
@jenniferlee-dsva Thank you for the update.   A couple of follow-up questions: 

1) This legacy page with the 2019-2020 school year list is already redirected
- Legacy page: www.benefits.va.gov/gibill/yellow_ribbon/yrp_list_2019.asp
- Redirects to: https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/#does-my-school-participate-in-

When the tool launches, this redirect will still need to be updated to redirect the user to the tool rather than this content page since that content will be going away.   We can keep this issue alive for that, and I will update it once confirmed. 

2)  Are we keeping the the individual state legacy pages up for the 2019-2020 year?  The above legacy page is the primary page for this school year, but there are 50+ child pages as well.  
Example: https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/al.asp

**Comment by stephorkandatx** [comment]
Jen to validate what, if anything, is needed

**Comment by jenniferlee-dsva** [comment]
This ticket has a lot of comments, so not sure where to start digging in. 

**Noting confirmation here after call w/YR business on 5/1.** 

1. Will will  NOT redirect existing YR historic years pages. (The business currently uses these for internal purposes.)
2. We will also NOT link to historic school information from thetool or from our YR content page. 
3. We will redirect their CURRENT year pages  for 2019-2020 (the URLs that @AnneHurley provided above) to the tool when tool release is complete. (Technically this isn't redirecting historic year info, but duplicative **_current_** year info that will now be provided bythe tool.)

Details here: https://dsva.slack.com/archives/C52CL1PKQ/p1588348663204400?thread_ts=1588347896.203200&cid=C52CL1PKQ

**Comment by stephorkandatx** [comment]
Description of ticket updated w/ the URLs previously provided in comments + clarification that they will redirect to the tool. This work is queued but will remain a to-do item until launch day when the tool is released to production.

**Comment by stephorkandatx** [comment]
This ticket will be removed from Sprint 20. It is included in our release plan as our "Go Live" step. We will bring this into the sprint when we receive Staging Review approval and schedule a date for Phase 2 of the release plan.

**Comment by stephorkandatx** [comment]
Blocking this until it's ready to play per the release plan

**Comment by stephorkandatx** [comment]
@selina we will need to send a notification out, for these redirects. Please queue this but send to Jen Lee directly. She will add additional notes to the notification about the release and next steps.

**Comment by lunascoop** [comment]
@stephorkandatx I am scheduling these redirects for **05/28**. Sending out notification to Jen directly now.

**Comment by mnorthuis** [comment]
I need this ticket to go back through me since there has been updates to where redirects are going.  I will need to review and  update documentation on my end.  Will review this within the next couple of days.

**Comment by stephorkandatx** [comment]
Thanks @mnorthuis we will wait for your review/signoff before this moves forward

**Comment by mnorthuis** [comment]
I added one URL to the list above - www.benefits.va.gov/gibill/yellow_ribbon/yrp_list_2019.asp
- this legacy page is already redirected to an anchor point on the current content page that previously had all the YR school info, now that that info has moved the tool, this page should also redirect directly to the tool.   

All the above URLs are good to go.  You may be able to implement a single greedy redirect that captures everything after  https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/.

**Comment by stephorkandatx** [comment]
Thanks @mnorthuis !

**Comment by kelsonic** [comment]
Reposting @ncksllvn's comment on the PR:

>Just wondering - do we, in the end, want to redirect the entire www.benefits.va.gov/gibill/yellow_ribbon/ directory?

**Comment by jenniferlee-dsva** [comment]
@kelsonic @stephorkandatx CC @mnorthuis - Please do NOT redirect the entire` /yellow_ribbon/ `directory. There are pages under that directory which are for school administrators and need to continue to be available for schools. 

But per Mikk's message here, we can redirect the 2019 directory after "/states" -- `https://www.benefits.va.gov/gibill/yellow_ribbon/2019/states/`

We are not redirecting any prior years.

**Comment by lunascoop** [comment]
Redirects are live and validated. Closing ticket and updating stakeholders.
